Scripts/Pointer/ObjectPointer.cs

```diff
@@ -311,6 +314,7 @@ protected virtual void OnEnable()
         protected virtual void OnDisable()
         {
             Deactivate();
+            Deactivated?.Invoke(HoverTarget, this);

```

Comment:
The `Deactivate` method is doing more than just emitting this event. I wonder if we need anything like the `TryEmitExit` its doing (among the other things) in here, too?